conda-forge.yml

```
@@ -6,13 +6,11 @@ build_platform:
   osx_arm64: osx_64
 conda_build:
   pkg_format: '2'
+conda_build_tool: "conda-build+conda-libmamba-solver"
```

```suggestion
conda_build_tool: "conda-build"
conda_install_tool: "conda"
conda_solver: "libmamba"
```

Let's try this. Looks like Windows is stuck on conda 23.3 for some reason.

---

That seems to have gone worse than before - now osx is also failing.